data/gdp.py

```diff
@@ -11,11 +11,75 @@
 from dataclasses import dataclass
 import warnings
 
-folder = "../data/raw/gdp-v2.00/"
+
+def parse_directory_file(filename: str) -> pd.DataFrame:
+    """
+    Read a directory file
+
+    Note: due to naming of those files, it requires manual intervention to update the last file name after an update of the dataset
+
+    Args:
+        filename (str): filename of the dirfl file
+
+    Returns:
+        pd.DataFrame: sorted list of drifters
+    """
+    aoml_dirfl_url = "https://www.aoml.noaa.gov/ftp/pub/phod/buoydata/"
+
+    df = pd.read_csv(join(aoml_dirfl_url, filename), delimiter="\s+", header=None)
+    df[4] = df[4] + " " + df[5]
+    df[8] = df[8] + " " + df[9]
+    df[12] = df[12] + " " + df[13]
```

Comment:
```suggestion
    df[4] += " " + df[5]
    df[8] += " " + df[9]
    df[12] += " " + df[13]
```

Can you also write a short comment above these three lines explaining why we do this?